plugins/link/dialogs/anchor.js

```diff
@@ -136,10 +136,23 @@ CKEDITOR.dialog.add( 'anchor', function( editor ) {
 				label: editor.lang.link.anchor.name,
 				required: true,
 				validate: function() {
-					if ( !this.getValue() ) {
+					// W3C HTML 5.2 Recommendation; §3.2.5:
```

Comment:
The HTML 5.2 specification does not longer exist. Could you update the reference to point to https://html.spec.whatwg.org/multipage/dom.html#global-attributes ?